hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerPrepare.java

```diff
@@ -202,7 +201,7 @@ public void testPrepareWithRestart() throws Exception {
     // modified cluster.
     shutdown();
     init();
-    setup();
+    initInstanceVariables();
```

Comment:
Why do we need to call `initInstanceVariables` here again if we are using `@BeforeEach`?

Reply:
This was the old code:

```
// Create fresh cluster for this test to prevent timeout from restarting
// modified cluster.
shutdown();
init();
setup();

... (test)
```

Basically it re-runs `@AfterAll` and `@BeforeAll` manually for this test case, but then does not call the `@BeforeEach` method, rather only a specific part of it.  Normally the code in `@BeforeEach` stores `cluster` etc. in instance variables, _and_ performs other steps.  I guess we want to avoid those other steps for this specific test case.

The only change in this test case is that `TestOzoneManagerPrepare.setup()` was renamed to `initInstanceVariables()` because it conflicted with the new `setup()` method from parent class.

Reply:
I found https://github.com/apache/ozone/pull/2689/files#r718204488 and it says it is done this way to prevent CI from timing out. Though I fully don't understand it how.

Since your change is just changing method name, it is OK to leave as it.